### PR TITLE
Updated the main README

### PR DESCRIPTION
**Pull request by Engineering-Robert**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -15,13 +15,17 @@ Run the following commands:
...
 
 ## Development
-You can use `yarn watch` to automatically recompile assets when you change them.
-You'll need docker compose.
+
+### Compiling Assets
+You can use `yarn watch` to automatically recompile assets when you change them. You'll need *Docker Compose* for this to work (*Docker Compose* comes pre-installed with **Docker for Mac** and **Docker Toolbox**).
+
+### Local Environment Setup
+You'll need to create your own `.env` file in the root project directory. Copy the `.env.example` file and rename it as your `.env` file. In order to view API information related to related to meetup.com, you'll need to [generate a new API key](https://secure.meetup.com/meetup_api/key/) and past the key in your `.env` file as the value for `MEETUP_API_KEY`.
```
Comment (Voziv) [inline readme.md]: 'and past the key' 

Should be `paste` instead of `past`

File: readme.md [hunk 2]
```diff
@@ -15,13 +15,17 @@ Run the following commands:
 - `composer install`
 - `yarn install`
 - `yarn dev`
-- `cp .env.example .env` (edit this file based on your configuration if you need to setup things like a database)
+- `cp .env.example .env`
```
Comment (Voziv) [inline readme.md]: Probably can leave this out if we're covering it under the local environment setup

File: readme.md [hunk 3]
```diff
@@ -15,13 +15,17 @@ Run the following commands:
...
 - `php artisan key:generate`
 - `docker-compose up -d`
 
 ## Development
-You can use `yarn watch` to automatically recompile assets when you change them.
-You'll need docker compose.
+
+### Compiling Assets
+You can use `yarn watch` to automatically recompile assets when you change them. You'll need *Docker Compose* for this to work (*Docker Compose* comes pre-installed with **Docker for Mac** and **Docker Toolbox**).
```
Comment (Voziv) [inline readme.md]: `yarn watch` is totally unrelated to `docker-compose`

File: readme.md (hunk 1 above)
Comment (Engineering-Robert) [reply readme.md]: Grammar nazi

File: readme.md (hunk 2 above)
Comment (Engineering-Robert) [reply readme.md]: Ok

File: readme.md (hunk 1 above)
Comment (Voziv) [reply readme.md]: Spelling*

File: readme.md (hunk 3 above)
Comment (Engineering-Robert) [reply readme.md]: Oopsies

**Comment by Engineering-Robert** [comment]
@Voziv Added your requested changes.

**Pull request merged**